src/dialog.jsx

```diff
@@ -46,7 +47,7 @@ const TransitionItem = React.createClass({
     this.setState({muiTheme: newMuiTheme});
   },
 
-  componentWillEnter(callback) {
+  componentDidAppear(callback) {
```

Comment:
Yes - this looks like a mistake on my part (though it doesn't seem to have any impact)